src/ng/directive/ngShowHide.js

```diff
@@ -98,7 +98,7 @@ var NG_HIDE_IN_PROGRESS_CLASS = 'ng-hide-animate';
  * @example
   <example module="ngAnimate" deps="angular-animate.js" animations="true">
     <file name="index.html">
-      Click me: <input type="checkbox" ng-model="checked"><br/>
+      Click me: <input type="checkbox" ng-model="checked" aria-label="toggle ngHide and ngShow"><br/>
```

Comment:
Is this actually what's happening ("toggle ngHide and ngShow")? It seems like this is actually toggling `.ng-hide`. @gkalpak can you weigh in?